client/app/host/TeamInspector.tsx

```diff
@@ -8,16 +9,16 @@ import {
     SelectValue,
 } from '@/components/ui/select';
 import { ArrowLeft, Wifi, WifiOff } from 'lucide-react';
-import { useTeams, useSelectedTeam, useSelectedTeamIdx } from '@/lib/host-state';
+import { teamsAtom, selectedTeamAtom, selectedTeamIdxAtom } from '@/lib/host-state';
 import { Card, CardHeader, CardTitle } from '@/components/ui/card';
 import TeamInfo from './TeamInfo';
 import { Button } from '@/components/ui/button';
 import { Separator } from '@/components/ui/separator';
 
 export default function TeamInspector() {
-    const { teamList } = useTeams();
-    const { selectedTeam } = useSelectedTeam();
-    const { setSelectedTeamIdx } = useSelectedTeamIdx();
+    const [teamList, _setTeamList] = useAtom(teamsAtom);
+    const [selectedTeam, _setSelectedTeam] = useAtom(selectedTeamAtom);
+    const [_selectedTeamIdx, setSelectedTeamIdx] = useAtom(selectedTeamIdxAtom);
```

Comment:
In this case, since you don't need to get the value, you can just do this:
```suggestion
    const setSelectedTeamIdx = useSetAtom(selectedTeamIdxAtom);
```
(This applies to all other usages of `useAtom` in this way)